_data/guides.yml

```diff
@@ -189,5 +201,21 @@ toc:
 
 - title: Administering Federation
   section:
-  - /docs/admin/federation/kubfed/
-  - docs/admin/federation/index.md
+  - docs/admin/federation/kubefed.md
+  - title: Federated Kubernetes Objects
+    section:
+    - docs/user-guide/federation/index.md
+    - docs/user-guide/federation/configmap.md
+    - docs/user-guide/federation/daemonsets.md
+    - docs/user-guide/federation/deployment.md
+    - docs/user-guide/federation/events.md
+    - docs/user-guide/federation/federated-ingress.md
+    - docs/user-guide/federation/namespaces.md
+    - docs/user-guide/federation/replicasets.md
+    - docs/user-guide/federation/secrets.md
+    - docs/federation/api-reference/README.md
```

Comment:
I'll try it. I'm leery of renaming things to "index.md" with whatever weird URL-shortening/prettifying has been installed on the Kubernetes website. I'll give it a test run and see if it works.